sbpy/utils/tests/test_decorators.py

```diff
@@ -0,0 +1,22 @@
+from warnings import catch_warnings
+import pytest
+from ..decorators import requires, optional
+from ...exceptions import RequiredPackageUnavailable, OptionalPackageUnavailable
+
+
+def test_requires():
+    @requires("unavailable_package")
+    def f():
+        pass
```

Comment:
Separate function out of `test_requires` and add a second test to cover this line.